README.md

```diff
@@ -53,3 +53,16 @@ will produce a charm that looks like
             └── ...
 
 which should be all you need to `juju deploy` the charm!
+
+## How do I run it from source?
+
+    git clone https://github.com/canonical/charmcraft.git
+    virtualenv venv
+    . venv/bin/activate
+    pip install -r requirements.txt
+    PYTHONPATH=.:${PYTHONPATH} python charmcraft/main.py
```

Comment:
`python -m charmcraft` is shorter and more to the point (no need to fix paths, or call internal modules)